src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/IPropertyExtensions.cs

```diff
@@ -28,12 +28,12 @@ internal static IPropertyMetadata ToPropertyMetadata(this IProperty property, Ty
             propertyMetadata.IsForeignKey = property.IsForeignKey();
 
             propertyMetadata.IsEnum = property.ClrType.GetTypeInfo().IsEnum;
-            propertyMetadata.IsReadOnly = property.IsReadOnlyBeforeSave;
+            propertyMetadata.IsReadOnly = property.BeforeSaveBehavior != PropertyValueBehavior.UseValue;
```

Comment:
What is IsReadOnly used for? EF will never set this by default for any property and it would be very uncommon for app code to set it. (Same is true for IsReadOnlyBeforeSave.)

That being said, this is the correct change to preserve the behavior from before.